.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 npm-debug.log
 server/db
 server/assets/*.map
+server/temp
```

Comment:
Should be `/server/temp`.

Otherwise, it will ignore every `server/temp` files/folder on the project.

It should be the same for the rest of the file BTW.